src/modules/handle-tweet.ts

```diff
@@ -81,10 +82,35 @@ const events = {
   },
 };
 
-const mainNetworks = ["bepro", "taikai"];
+export const mainNetworks = ["bepro", "taikai"];
 process.env.NETWORK_NAME &&
   mainNetworks.push(process.env.NETWORK_NAME.toLowerCase());
 
+export async function dispatchTweets(
+  bounties: BountiesProcessed,
+  entity: string,
+  event: string,
+  networkName: string
+) {
+  if (
+    !bounties ||
+    !mainNetworks.includes(networkName.toLocaleLowerCase()) ||
+    !events?.[entity]?.[event]
+  )
+    return;
+
+  const createTweet = Object.values(bounties).map(
```

Comment:
This must be made _inside_ the `Promise.all` or you wont be taking advantage of the `Promise.all`

```ts
return Promise.all( Object.values( ....
```

Also, _don't_ await inside a promise all -- you want them all to go at the same time, you don't want to await on each loop for the previous to have finished